metric-publishers/emf-metric-logging-publisher/src/main/java/software/amazon/awssdk/metrics/publishers/emf/EmfMetricLoggingPublisher.java

```diff
@@ -187,11 +184,14 @@ public Builder metricCategories(MetricCategory... metricCategories) {
         }
 
         /**
-         * Configure the LogGroupName key that will be put into the emf log to this publisher. This is a required key for
-         * using the CloudWatch agent to send embedded metric format logs that tells the agent which log group to use.
+         * Configure the LogGroupName key that will be put into the emf log to this publisher. This is required when using
+         * the CloudWatch agent to send embedded metric format logs that tells the agent which log
+         * group to use.
          *
-         * <p> This field is required and must not be null or empty.
-         * @throws NullPointerException if logGroupName is null
+         * <p> If this is not specified, for AWS lambda environments, {@code "/aws/lambda/" + "AWS_LAMBDA_LOG_GROUP_NAME"}
```

Comment:
nit, it's just `AWS_LAMBDA_LOG_GROUP_NAME` and not `aws/lambda/ + AWS_LAMBDA_LOG_GROUP_NAME`, right?